src/test/java/edu/hm/mschober/ArrayListTest.java

```diff
@@ -0,0 +1,17 @@
+package edu.hm.mschober;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Michael Schober, mschober@hm.edu
+ * @version 2020-31-03
```

Comment:
Sinnvoll ist ein Text, der den Test beschreibt. Die Version ist nicht hilfreich, die steht ja in Git.